lib/testing/src/lib/core/pages/form/start-form.page.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { element, by } from 'protractor';
+import { BrowserVisibility } from '../../utils/browser-visibility';
+
+export class StartFormPage {
+    startProcessForm = element(by.css('.adf-start-form-container'));
+
+    async checkStartProcessFormIsDisplayed(): Promise<void> {
+        await BrowserVisibility.waitUntilElementIsVisible(this.startProcessForm);
```

Comment:
can you make startProcessForm a testelement has already that method inside

Reply:
I've changed `startProcessForm` locator to TestElement